Remotion/Core/Core/Reflection/TypeDiscovery/BaseTypeCache.cs

```diff
@@ -69,13 +70,15 @@ public static BaseTypeCache Create (IEnumerable<Type> types)
             } while (nextBaseType != null);
           }
 
-          foreach (var interfaceType in type.GetInterfaces())
+          processedInterfaces.Clear();
+          foreach (var implementedInterface in type.GetInterfaces())
           {
-            var normalizedInterfaceType = GetNormalizedType(interfaceType);
+            var normalizedInterfaceType = GetNormalizedType(implementedInterface);
```

Comment:
last question: change `implementedInterface` back to `interfaceType` to reduce change set?